frontend/src/components/Collections/components/Grid/components/Row/DatasetRow/components/MoreDropdown/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { Position } from "@blueprintjs/core";
```

Comment:
This component utilizes the new common component `src/components/common/MoreDropdown`